nodes/src/nodes/tool_bland_ai/IGlobal.py

```
@@ -0,0 +1,85 @@
+    driver: BlandDriver | None = None
+
+    def beginGlobal(self) -> None:
+        if self.IEndpoint.endpoint.openMode == OPEN_MODE.CONFIG:
+            return
+
+        cfg = Config.getNodeConfig(self.glb.logicalType, self.glb.connConfig)
+
+        api_key = str(cfg.get("apikey") or "").strip()
+        if not api_key:
+            warning("Bland AI API key is required. Get one at https://www.bland.ai")
+            raise ValueError("Bland AI API key not configured")
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)